Britannia/events/BTWK_storyline_events.txt

```diff
@@ -4,96 +4,676 @@
+# BTWK_story_event.0110 = Uther is Dead - Global Event
+# BTWK_story_event.0111 = Uther is Dead - Tag Switch Event for Player Mordred
+# BTWK_story_event.0112 = Arthur Introductory Event
+
+## AFTERMATH OF UTHER'S DEATH ##
+# BTWK_story_event.0113 = Event for Powysian/Gwentish war - triggered by decision
+
+
+### PART 2 - THE PRINCESS BRIDE ###
+## OWAIN/ARTHUR POWER STRUGGLE ##
+# BTWK_story_event.0201 = Some time after betrayal/after Uther's death Cadwy sends offer to Owain
+
+### PART 3 - THE RETURN OF MERLIN ###
+
+### PART 4 - THE ISLE OF THE DEAD ###
+
+### PART 5 - THE SHIELD WALL ###
+
+### PART 6 - THE DARK ROAD ###
+
+### PART 7 - THE BROKEN ROAD ###
+
+### PART 8 - CAMELOT ###
 
-### ARTHUR REGENCY CHAIN ###
-# BTWK_story_event.0008 = Regent decides to give power to Arthur
-# BTWK_story_event.0009 = Arthur accepts regency
-# BTWK_story_event.0010 = Notify Mordred
-# BTWK_story_event.0011 = Notify other vassals
-# BTWK_story_event.0012 = Set Flag
+### PART 9 - THE MYSTERIES OF ISIS ###
 
+### PART 10 - THE FIRES OF MAI DUN ###
 
-############## UTHER'S DEATH and ARTHUR'S RETURN ##############
+### PART 11 - MYNYDD BADDON ###
 
+### PART 12 - NIMUE'S CURSE ###
 
-BTWK_story_event.0001 = {  # Triggered by on_game_start on_action.
+### PART 13 - THE LAST ENCHANTMENT ###
+
+
+############## UTHER'S ILLNESS and ARTHUR'S RETURN ##############
+
+BTWK_story_event.0101 = {
 	type = character_event
-	title = BTWK_story_event.0001.t
-	desc = BTWK_story_event.0001.desc
-	# More fitting music cue/sounds needed
-	theme = skull
-	# Won't use "window = fullscreen_event" until figure out how to add portraits to the event.
+	title = BTWK_story_event.0101.t
+	desc = BTWK_story_event.0101.desc
+	theme = realm 
+
+	right_portrait = {
+		character = character:1  # Uther
+		animation = personality_honorable
+	}	
+
+	option = {
+		name = BTWK_story_event.0101.a
+		character:1 = {
+			trigger_event = {
+				id = BTWK_story_event.0103
+				days = 50
+			}
+		}
+
+		every_ruler = {
+			if = {
+				limit = {
+					AND = {
+						holds_any_storyline_britannia_kingdoms = yes 
+						culture = {
+							culture_is_brythonic_culture_trigger = yes
+						}
+					}
+				}
+				trigger_event = {
+					id = BTWK_story_event.0102
+					days = 25
+				}
+			}
+		}
+
+	}
+}
+
+BTWK_story_event.0102 = { # Event for ai selection of whether they want to go to glevum + catalogue norwenna's suitors
+
+# MAY WANT A SECOND GLOBAL VARIABLE LIST FOR ALL VALID KINGS, IN CASE WE MAKE GLEVUM INTO A RESKINNED FEAST ACTIVITY
+# IN THIS CASE, PROBABLY WANT TO REFERENCE NEW FILE SO I DON'T HAVE TO KEEP CHANGING EVENT ID - "BTWK_storyline_activity_events", etc.
+# AI CHANCE MUST STILL BE MODIFIED ACCORDINGLY
+# ADD AN OPINION BASED MODIFIER TO AI CHANCE?
+
+	type = character_event
+	title = BTWK_story_event.0102.t
+	desc = BTWK_story_event.0102.desc
+	theme = realm 
+
+	right_portrait = {
+		character = root  
+		animation = personality_honorable
+	}
+	
+	option = {  # Go to the council
```

Comment:
Maybe there should be a separate tooltips or options depending on whether or not you or your heir is eligible to become Mordred's guardian? I guess it might be possible that you want to attend but don't want to take responsibility over Mordred or something.